src/gps/cas.h

```diff
@@ -61,3 +61,26 @@ static const uint8_t _message_CAS_CFG_NAVX_CONF[] = {
     0x00, 0x00, 0x00, 0x00, // Time Accuracy Max
     0x00, 0x00, 0x00, 0x00  // Static Hold Threshold
 };
+
```

Comment:
Since this is almost identical to the one before, you might want to consider changing just the mode value in the setup. 

See https://github.com/meshtastic/firmware/pull/6630/files#diff-c9e9d981789c05f2633c8bc91527d5c33f714ea43d106684250598ec0d4487ccL587 where the vehicle mode is changed to stationary.